### PR TITLE
Document that model_locations/dir now supports glob

### PR DESCRIPTION
**Pull request by mfn**
Feature was added with https://github.com/barryvdh/laravel-ide-helper/pull/921

### CODE REVIEW

File: config/ide-helper.php [hunk 1]
```diff
@@ -89,6 +89,9 @@
     | Define in which directories the ide-helper:models command should look
     | for models.
     |
+    | glob patterns are supported to easier reach models in sub-directories,
+    | e.g. `app/Services/* /Models` (without the space)
```
Comment (barryvdh) [inline config/ide-helper.php:93]: So why the space?

Comment (mfn) [reply config/ide-helper.php:93]: Yeah well :) because otherwise it would end the phpdoc 💥

Comment (mfn) [reply config/ide-helper.php:93]: And I tried (maybe too hard) to give a useful example

Comment (barryvdh) [reply config/ide-helper.php:93]: hehe good point.

**Pull request merged**